README.md

```diff
@@ -14,7 +14,11 @@ The user cases and documentation of the Fornax Initiative are currently availabl
 ## Content contributing
 
 In this repository, we follow the standard practice of the Scientific Python ecosystem and use Jupytext and MyST Markdown Notebooks.
-You will need ``jupytext`` installed for your browser to recognise the markdown files as notebooks (see more about the motivation and technicalities e.g. here: https://numpy.org/numpy-tutorials/content/pairing.html).
+
+Please visit [the upstream documentation](https://mystmd.org/guide/md-vs-ipynb) to learn more about the reasoning behind the choice.
+However, we add it here that you can easily have the same user experience in JupyterLab if these two dependencies are installed:
```

Comment:
please clarify what 'it' refers to in this line.  .ipynb functionality?

Reply:
it's the "note" that we make, I've rephrased.